staff/views/applications.py

```diff
@@ -145,3 +145,22 @@ def get_queryset(self):
             qs = qs.filter(status=status)
 
         return qs
+
+
+@method_decorator(require_role(CoreDeveloper), name="dispatch")
+class ApplicationRemove(View):
+    def post(self, request, *args, **kwargs):
```

Comment:
bug(possible): Deleting from the Staff Area gives me a 405 (Method Not Allowed) error. Presumably this is because the [template has the Delete button](https://github.com/opensafely-core/job-server/pull/1474/files#diff-3ea9c38b376f0d215a1c2820c05570465bb3dede15279456963b3b1a5c3949a5R78) as a simple `href`.